pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -215,7 +215,7 @@ func (m *MultishareOpsManager) startInstanceWorkflow(ctx context.Context, w *Wor
 	// 1. verify no instance ops or share (belonging to the instance) ops running for the given instance.
 	// 2. Start the instance op.
 	if w.instance == nil {
-		return nil, status.Errorf(codes.Internal, "instance not found in workflow object")
+		return nil, status.Errorf(codes.InvalidArgument, "instance not found in workflow object")
```

Comment:
this would be internalError (ihandled by StatusForError()) since we do not expect the instance object to be nil here?